style.css

```diff
@@ -0,0 +1,136 @@
+body {
+  font-family: poppins, sans-serif;
+}
+
+.parent-container {
+  display: flex;
+  position: absolute;
+  height: 718px;
+  width: 375px;
+}
+
+.image {
+  position: static;
+  margin-top: 130.85px;
+  width: 375px;
+  height: 587.15px;
+  margin-left: -10px;
+}
+
+.header-section {
+  display: flex;
+  position: absolute;
+  flex-direction: row;
```

Comment:
- [ OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  flex-direction: row;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉